packages/smooth_app/lib/pages/product/add_new_product_helper.dart

```diff
@@ -50,7 +50,10 @@ class AddNewProductTitle extends StatelessWidget {
   @override
   Widget build(BuildContext context) => Text(
         label,
-        style: Theme.of(context).textTheme.displaySmall,
+        style: const TextStyle(
+          fontSize: 18.0,
+          fontWeight: FontWeight.bold,
+        ),
```

Comment:
That wont look good on small devices, please use 

Theme.of(context).textTheme.displaySmall

or another one which is most fitting and edit it with `.copyWith()`